staging/src/k8s.io/legacy-cloud-providers/vsphere/OWNERS

```diff
@@ -1,17 +1,16 @@
 # See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
Did we want to add 
# We are no longer accepting features into k8s.io/legacy-cloud-providers.
# Any kind/feature PRs must be approved by SIG Cloud Provider going forward.

here as well?